pkg/forklift-api/webhooks/mutating-webhook/mutators/ovirt-cert-mutator.go

```diff
@@ -76,6 +86,10 @@ func (mutator *OvirtCertMutator) Mutate(ar *admissionv1.AdmissionReview) *admiss
 			log.Info("Engine CA certificate was missing, updating the secret")
 		}
 
+		if !insecure {
+			secret.Data["insecureSkipVerify"] = []byte("false")
+		}
```

Comment:
we certainly don't need to check `if !insecure` here (since it was checked before) but why setting the secret - it should already be set this way, no?

Reply:
because later on  we are relaying on this value to set the transfer,  we don't want to assume that its already there.
I've changed the code so it only be added if not existing at all.